src/models/dish.js

```diff
@@ -5,22 +5,29 @@ const dishSchema = new mongoose.Schema({
   userId: { type: mongoose.Schema.Types.ObjectId, ref: 'User' },
   name: {
     type: String,
-    required: true,
+    required: false,
   },
   description: {
     type: String,
-    required: true,
+    required: false,
   },
   price: {
     type: String,
-    required: true,
+    required: false,
   },
   /* picture: {
     type: String,
-    required: true,
+    required: false,
```

Comment:
If you make those false, that means the chef will be able to create a dish without giving the a name or description or even price, in our case now the chef is able to create an empty dish, and this dish will just have the `chefId and userId`
We need to return them to how they was.